smart_open/ssh.py

```diff
@@ -43,18 +44,22 @@
 DEFAULT_PORT = 22
 
 
-def _connect(hostname, username, port):
+def _connect(hostname, username, port, rsakey = None):
```

Comment:
Get rid of spaces around keyword parameter, here and everywhere.

```suggestion
def _connect(hostname, username, port, rsakey=None):
```